GothicLlamasAPI/pom.xml

```diff
@@ -68,10 +68,21 @@
             <artifactId>dropwizard-jdbi3</artifactId>
             <version>2.0.23</version>
         </dependency>
+<!--        <dependency>-->
+<!--            <groupId>org.jdbi</groupId>-->
+<!--            <artifactId>jdbi3-core</artifactId>-->
+<!--            <version>3.0.1</version>-->
+<!--        </dependency>-->
```

Comment:
We don't want commented out code in the codebase. It isn't lost if you delete it because of git.